fe/fe-core/src/main/java/org/apache/doris/nereids/rules/expression/rules/OrToIn.java

```diff
@@ -57,7 +55,7 @@
  * adding any additional rule-specific fields to the default ExpressionRewriteContext. However, the entire expression
  * rewrite framework always passes an ExpressionRewriteContext of type context to all rules.
  */
-public class OrToIn extends DefaultExpressionRewriter<OrToInContext> implements
+public class OrToIn extends DefaultExpressionRewriter<ExpressionRewriteContext> implements
```

Comment:
`OrToInContext` is useless? Can we remove it?

Reply:
have removed